esy-build-package/Config.re

```diff
@@ -3,8 +3,10 @@ module Store = EsyLib.Store;
 [@deriving (show, to_yojson)]
 type t = {
   projectPath: EsyLib.Path.t,
+  globalStorePrefix: EsyLib.Path.t,
   storePath: EsyLib.Path.t,
   localStorePath: EsyLib.Path.t,
+  disableSandbox: bool,
 };
```

Comment:
Is there a way we could keep only three paths there — project path, path for immutable builds (global store) and path for transient builds (local store)?

I think here we could determine `storePath` out of `globalStorePrefix`, right?

Reply:
Yeah, we can definitely do that. I'll be on it.